doc/build/changelog/unreleased_14/6990.rst

```diff
@@ -0,0 +1,8 @@
+.. change::
+    :tags: usecase, engine, orm
+    :tickets: 6990
+
+    This change adds a scalars() method to the connection and session classes
```

Comment:
**mike bayer** (zzzeek) wrote:

I have new text for this:

    Added new methods :meth:`_orm.Session.scalars`,
    :meth:`_engine.Connection.scalars`, :meth:`_asyncio.AsyncSession.scalars`
    and :meth:`_asyncio.AsyncSession.stream_scalars`, which provide a short cut
    to the use case of receiving a row-oriented :class:`_result.Result` object
    and converting it to a :class:`_result.ScalarResult` object via the
    :meth:`_engine.Result.scalars` method, to return a list of values rather
    than a list of rows. The new methods are analogous to the long existing
    :meth:`_orm.Session.scalar` and :meth:`_engine.Connection.scalar` methods
    used to return a single value from the first row only. Pull request
    courtesy Miguel Grinberg.

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/3065